quicktests/umd/lib/require.js

```diff
@@ -7,6 +7,7 @@
 //problems with requirejs.exec()/transpiler plugins that may not be strict.
 /*jslint regexp: true, nomen: true, sloppy: true */
 /*global window, navigator, document, importScripts, setTimeout, opera */
+/*eslint-disable */
```

Comment:
Modifying the library? Is there a way to tell ESLint to ignore the directory instead?